contents/tutorials/churn-rate.md

```diff
@@ -0,0 +1,87 @@
+topics: ['trends']
+---
+
+- **Level:** Easy 🦔
+- **Estimated reading time:** 6 minutes ☕️
+
+The word churn once reminded people of fresh butter, now it brings thoughts of lost users and revenue to product and marketing teams.
+
+The churn rate is the percentage of users who stop an action during a certain period. It is used to understand the negative changes in usage, revenue, or other growth metrics. Understanding churn rate is key to figuring out problem areas in your product. 
+
+In this tutorial, we’ll calculate and visualize the churn rate then use PostHog’s features of session recordings, cohorts, and actions to lower it.
+
+### Prerequisites
+
+To follow this tutorial along, you need to have:
+
+1. **[Deployed PostHog](https://posthog.com/docs/deployment)**.
+
+## Calculating churn rate
+
+To calculate churn rate, first, you must define what ended or stopped. For subscription-based businesses, this could be monthly recurring revenue attributable to a subscriber. For a customer success or sales team, it could be organizations usage. For a product team, it might be product usage events by a user. For this tutorial, this product usage is what we’ll focus on.
+
+To calculate product usage churn, we must have a group of users, events from usage they are triggering, and a time period. We can look back (however long your time period is) to check the total number of users triggering events and compare it to the number triggering events now. Our churn rate is the number of users who stopped divided by the total number of users at the beginning of the period.
+
+> If we had 100 users who used triggered events for a feature at the beginning of the period, and 80 who were continuing to trigger events it at the end, we’d have a churn rate of 20% because (100 - 80) / 100 * 100 = 20%
+
+Better yet, we can visualize and refine this calculation. PostHog has tools, like retention and lifecycle charts, that help you understand churn rate and see its changes over time.
+
+## Visualizing churn rate
+
+To start visualizing churn rate, you can set up a retention funnel for users completing an action or event, then see how many continue their usage. The action or event you choose could be something like an autocapture of a pageview, an event capture you’ve written into your code or a combination of events (an action) that define feature usage. For us, we’ll set up a retention chart for unique users who completed the event `downloaded_file` over the last 8 weeks. 
+
+![Retention chart](../images/tutorials/churn-rate/retention-chart.png)
+
+We can calculate our weekly churn and total churn by comparing our initial cohort size with the percentage of users remaining. For example, if we care about weekly churn, we can compare Week 0 and Week 1. If we care about churn across the entire period, we can compare Week 0 and Week 8. To learn more about retention charts, check out our [product manual](https://posthog.com/manual/retention).
```

Comment:
```suggestion
We can calculate our weekly churn and total churn by comparing our initial cohort size with the percentage of users remaining. For example, if we care about weekly churn, we can compare Week 0 and Week 1. If we care about churn across the entire period, we can compare Week 0 and Week 8. To learn more about retention charts, check out our [product manual](/manual/retention).
```

We use relative paths for internal links as well. Not sure how important this actually is, but I'm told it's best practice so 🤷!

Reply:
ahh right